src/base/bittorrent/session.cpp

```diff
@@ -269,6 +288,8 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), m_isUTPEnabled, clampValue(0, 1))
```

Comment:
My 2 cents: long string constants (as comparing to short string constants like '0', '1', etc.) in a config file give more freedom when upgrading and replacing set of possible options. 

> 1. Neither numbers nor words are meaningless to the user, because there is no available information about it.

Partially disagree: user can't get their options for the setting, but at least can guess what the current value means.

> 2. If we had a user manual (or other source of information, eg. man), we could describe either numeric values or string values.

It is much easier to spot a mistake in documentation or outdated part of it with long descriptive string constants.  

>It's easier for users to edit the ini too, 2 key strokes at most for changing ChokingAlgorithm here.
With string literals, users might get typos, uppercase lowercase issues especially when the user is not fluent in English.

I seriously doubt that users with typing problems would want to edit config files manually.

All in all, I do not reveal a secret here saying that short constants are simpler and easier for one-time setup or use, while long ones offer advantages over them when considering maintainability and long-term support. Just as identifiers in source code.